src/configurations/destinations/http/ui-config.json

```diff
@@ -579,15 +579,15 @@
                     "type": "textInput",
                     "configKey": "to",
                     "label": "Key",
-                    "regex": "^\\$(?:\\.|(\\.(\\w+|\\*)|\\[\\d+\\]|\\[('\\w+'|\"\\w+\")\\]|\\[\\*\\]|\\.\\w+\\(\\))*)$",
+                    "regex": "^(?:$|\\$(?:\\.|(\\.(\\w+|\\*)|\\[\\d+\\]|\\[('\\w+'|\"\\w+\")\\]|\\[\\*\\]|\\.\\w+\\(\\))*)$)",
```

Comment:
what's this change for ?

Reply:
To allow empty mapping. David added a comment that empty mapping is a valid use case